pkg/k8s/watchers/pod.go

```diff
@@ -640,19 +635,19 @@ func (k *K8sWatcher) DeleteHostPortMapping(pod *slim_corev1.Pod, podIPs []string
 	return nil
 }
 
-func (k *K8sWatcher) updatePodHostData(pod *slim_corev1.Pod, podIPs []string) (bool, error) {
+func (k *K8sWatcher) updatePodHostData(pod *slim_corev1.Pod, podIPs []string) error {
 	if pod.Spec.HostNetwork {
-		return true, fmt.Errorf("pod is using host networking")
+		return fmt.Errorf("pod is using host networking")
```

Comment:
This one can be skipped, it's not a warning.

Reply:
I've moved the check out of `(*K8sWatcher).updatePodHostData` to avoid having to reintroduce the `skipped bool` again. This also saves the `podIPs` lookup in that case.